### PR TITLE
Update notification_smart_motion_event.yaml with Android Alert Once logic

### LINKED ISSUE #7: Android notifications trigger multiple alerts per detection
Environment:
- Home Assistant 2023.6.3
- Supervisor 2023.06.2
- Operating System 10.3
- Frontend 20230608.0 - latest

I'm using the latest released version of this blueprint.

On my Android devices including a Samsung Galaxy S23 Ultra, Samsung Galaxy S23+, and a Samsung Galaxy Watch5 Pro using the Home Assistant companion app, each instance of a smart detection triggers three sound/vibrate alerts for the same  detection:
- Once for the initial event
- A second time when a static camera image of the event is retrieved and updates the notification
- A third time when a camera GIF of the event is retrieved and updates the notification

My blueprint variables look like this: image
My notification group contains all the Android mobile devices I noted above.

Messing with my Android notification settings for the Smart Detection notification channel didn't help.

I did some further digging and discovered that HA supports the Android-only [Alert Once](https://companion.home-assistant.io/docs/notifications/notifications-basic/#alert-once) notification option. Alert Once appears to be made for situations like these. I believe that setting the notification option `alert_once: true` in the blueprint will correct this behaviour. I also believe it may be an option users will want to control in the blueprint, but can't think of a use case for this, so in the meantime I think implementation could enable Alert Once by default to keep things simple, and later parameterize it based on user feedback.

For more context, see [my posts from today](https://community.home-assistant.io/t/unifi-protect-blueprints/362604/73?u=dbsmith) in the Home Assistant Blueprints Exchange thread.

### PR DESCRIPTION
**Pull request by dbensmith**
Resolves #7.

Test environment:
- Home Assistant 2023.6.3
- Supervisor 2023.06.2
- Operating System 10.3
- Frontend 20230608.0 - latest

Test devices:
- Samsung Galaxy S23 Ultra
- Samsung Galaxy S23+
- Samsung Galaxy Watch5 Pro

Adds Alert Once logic to Android notifications so updates to a smart detection notification do not trigger additional alerts on Android devices. Alert Once requires notification tags to function, so the if/else logic only sets Alert Once to true if the user has set a value for tag.

Not tested on iOS. I do not have access to any iOS devices, though this setting is Android-only so the expected impact is nil.

### CODE REVIEW

**Comment by dbensmith** [comment]
I believe enabling Alert Once for Android users is a sensible default and optimal for most use cases.

If it isn't, or there are other use cases where being notified more than once is desirabe, we should turn this setting into a parameter configurable by blueprint users in the HA UI.

This PR does not introduce a parameter, but I support anyone who wishes to do so.

I will submit this PR for review in the meantime. Please let me know if I have done any of this incorrectly as I am not a stranger to GitHub, but it is my first PR to a public open source repo.

**Comment by outdooricon** [comment]
I've also been experiencing the multiple buzzes. This would be really nice to have @AngellusMortis

File: blueprints/automation/unifiprotect/notification_smart_motion_event.yaml [hunk 1]
```diff
@@ -192,6 +192,13 @@ action:
         # Android high prority
         ttl: 0
         priority: high
+        # Android alert once (requires tag to be set)
+        alert_once: |
+          {% if tag != "" %}
+            true
+          {% else %}
+            false
+          {% endif %}
```
Comment (AngellusMortis) [inline blueprints/automation/unifiprotect/notification_smart_motion_event.yaml:201]:
```suggestion
        alert_once: {{ (tag != "") | string | lower }}
```

Additionally, all of the other blueprints need the same change made.

Comment (dbensmith) [reply blueprints/automation/unifiprotect/notification_smart_motion_event.yaml:201]: Done, will update PR shortly

**Comment by dbensmith** [comment]
Updated across all three blueprints. @AngellusMortis please review again when you have time. Thanks!

File: blueprints/automation/unifiprotect/notification_smart_motion_event.yaml (hunk 1 above)
Comment (dbensmith) [reply blueprints/automation/unifiprotect/notification_smart_motion_event.yaml]: @AngellusMortis just sat down to test properly and the one-liner `alert_once: {{ (tag != "") | string | lower }}` broke the blueprint.

HA error is `Failed to load blueprint: invalid key: "{'(tag != "") | string | lower': None}"`.

The `alert_once` logic didn't need to evaluate the tag contents or casing, and [it must return boolean](https://companion.home-assistant.io/docs/notifications/notifications-basic/#alert-once). 

Were you looking to condense the logic into a one-liner?

Reverting to draft status in the meantime.

**Comment by AngellusMortis** [comment]
You marked this as draft again and never unmarked it. Was there something else?

**Comment by dbensmith** [comment]
> You marked this as draft again and never unmarked it. Was there something else?

@AngellusMortis thank you for following up. The change you suggested broke the notification logic this PR intended to correct.

I added more details and asked for feedback upthread in https://github.com/AngellusMortis/unifiprotect_blueprints/pull/8#discussion_r1382290318.

As it stands I believe the original PR code is good and in theory should work across blueprints. We should have others test and not just me.

Apologies if I marked it as draft incorrectly. I really appreciate all your hard work on this integration!

**Comment by dbensmith** [comment]
Rebased and re-proposed the alert_once logic change, this time across all three blueprints. Ready for review @AngellusMortis

**Pull request merged**